File: Week3/classes/lib/animal.js
// lib/animal.js

"use strict";

class Animal {
	constructor(name, noise) { 
	this.name = name;
	this.noise = noise;
};

makeNoise () {
	console.log( this.name + ' says: ' + this.noise);
};

shout () {
	console.log (this.name + ' shouts: ' + this.noise.toUpperCase());
};
}

module.exports = Animal;

File: Week3/classes/lib/animal.rb
# lib/animal.rb

class Animal
	def initialize(name, noise)
		@name = name
		@noise = noise
	end
	def make_noise
		puts "#{@name} says #{@noise}"
	end
	def shout
		puts "#{@name} shouts #{@noise.upcase}"
	end
end
